files/commonfiles/class/Utility.php

```
@@ -225,17 +225,15 @@ function xoopsFormValidate_donation() { var myform = window.document.donation;
-                        $GLOBALS['xoopsUser']->getVar('email'),
-                        \XOOPS_LICENSE_KEY,
-                        \mb_strtoupper($GLOBALS['xoopsModule']->getVar('dirname')),
-                        \mb_strtoupper($GLOBALS['xoopsModule']->getVar('dirname')) . ' ' . $GLOBALS['xoopsModule']->getVar('name')
-                    );
-                    break;
+            if ($key == 2) {
+                $donationform[$key] = \sprintf(
+                    $donationform[$key],
+                    $GLOBALS['xoopsConfig']['sitename'] . ' - ' . ('' != $GLOBALS['xoopsUser']->getVar('name') ? $GLOBALS['xoopsUser']->getVar('name') . ' [' . $GLOBALS['xoopsUser']->getVar('uname') . ']' : $GLOBALS['xoopsUser']->getVar('uname')),
+                    $GLOBALS['xoopsUser']->getVar('email'),
+                    \XOOPS_LICENSE_KEY,
```

PHPStan (2.1.46)

[error] Constant XOOPS_LICENSE_KEY not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)